dags/openshift_nightlies/manifest.yaml

```diff
@@ -133,3 +133,17 @@ platforms:
         config:
           install: rogcp/sdn.json
           benchmarks: data-plane.json
+  prebuilt:
+    versions: ["4.10"]
```

Comment:
We can't control openshift versions for prebuilts so we should probably just make this 4.x or remove this field entirely if we can. This key is only used for release streams and naming the dag so we can use whatever we want